packages/aws-cdk-lib/aws-lambda/lib/runtime.ts

```diff
@@ -284,4 +284,9 @@ export class Runtime {
       other.family === this.family &&
       other.supportsInlineCode === this.supportsInlineCode;
   }
+
+  // Used to construct default custom resource runtime by region
+  public static fromRegionDefaultString(name: string): Runtime {
```

Comment:
Should have "node" in the name probably

Reply:
removing this altogether actually since it's only used in one place noe.